bower.json

```diff
@@ -9,7 +9,7 @@
   ],
   "dependencies": {
     "o-colors": "^5.0.0",
-    "o-fonts": "^4.4.0",
+    "o-fonts": "^4.5.0",
```

Comment:
Would increasing the minimum version wanted count as a breaking change?

Reply:
No we've never treated this as breaking and regularly increase the minor and patch version of components